redbot/core/commands/help.py

```diff
@@ -212,6 +213,9 @@ async def format_command_help(
         signature = (
             f"`{T_('Syntax')}: {ctx.clean_prefix}{command.qualified_name} {command.signature}`"
         )
+        if help_settings.show_aliases and command.aliases:
+            alias_fmt = T_("Aliases") if len(command.aliases) > 1 else T_("Alias")
+            signature += f"\n`{alias_fmt}: {humanize_list(command.aliases)}`"
```

Comment:
You will need to update handling for non-embed help but as discussed in Discord, this can wait until we decide on the formatting.